modPrecompiled/src/org/aion/precompiled/contracts/ContractBase.java

```diff
@@ -0,0 +1,5 @@
+package org.aion.precompiled.contracts;
```

Comment:
missing the license (in case the class is kept)

Reply:
remove this class.